src/file_path.rb

```diff
@@ -148,6 +148,14 @@ def log
         '/var/log/metalware'
       end
 
+      def asset_template(type)
+        File.join('/var/lib/metalware/repo/assets', "#{type}.yaml")
```

Comment:
In this case, it probably would be clearer to go `type + '.yaml'`

Reply:
Also the `/v/l/m/repo` shouldn't be hard coded. You can refer to it using the `repo` method.